frontend/src/components/PassageMatches.tsx

```diff
@@ -5,21 +5,22 @@ type TProps = {
   document: TDocument;
   setShowPDF: (show?: boolean) => void;
   setPassageIndex: (index: number) => void;
+  activeIndex?: number;
 };
 
-const PassageMatches = ({ document, setShowPDF, setPassageIndex }: TProps) => {
+const PassageMatches = ({ document, setShowPDF, setPassageIndex, activeIndex }: TProps) => {
   return (
     <>
       {!document ? (
         <div className="w-full flex justify-center h-96">
           <Loader />
         </div>
       ) : (
-        <div className="px-6">
+        <>
           {document.document_passage_matches.map((item, index: number) => (
             <div
               key={item.text_block_id}
-              className="py-4 cursor-pointer"
+              className={`p-4 cursor-pointer border hover:bg-offwhite ${activeIndex === index ? "border-blue-200 bg-lightgray" : "border-transparent"}`}
```

Comment:
might be worth naming these css classes in their own right e.g. `.document-index-active` / `document-index-inactive` ?